apps/test/unit/code-studio/pd/foorm/FoormEditorTest.js

```diff
@@ -0,0 +1,325 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {expect} from '../../../../util/reconfiguredChai';
+import {assert} from 'chai';
+import {
+  stubRedux,
+  restoreRedux,
+  getStore,
+  registerReducers
+} from '@cdo/apps/redux';
+import {Provider} from 'react-redux';
+import FoormEditor from '../../../../../src/code-studio/pd/foorm/FoormEditor';
+import foorm, {
+  setFormData
+} from '../../../../../src/code-studio/pd/foorm/editor/foormEditorRedux';
+import sinon from 'sinon';
+
+global.$ = require('jquery');
+
+describe('FoormEditor', () => {
+  let defaultProps, store, server;
+  beforeEach(() => {
+    stubRedux();
+    registerReducers({foorm});
+
+    server = sinon.fakeServer.create();
+    server.respondWith('POST', '/api/v1/pd/foorm/form_with_library_items', [
```

Comment:
Since some of the tests below have their own respondWith and some don't I think it might be more clear if this was included in each test individually

Reply:
all the tests need this api called stubbed out since it's called on page load

Reply:
Ah ok I see. Thanks for clarifying!